sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Microsoft.Azure.WebJobs.Extensions.SignalRService.csproj

```diff
@@ -11,24 +11,24 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!--Pin the versions of packages that may be upgraded during the process of Azure SDK moving all dependencies to the 8.x line to avoid incompatibility, until a compatible dependency gragh is figured out and finalized.-->
     <PackageReference Include="MessagePack" />
-    <PackageReference Include="Microsoft.Extensions.Azure" />
-    <PackageReference Include="Microsoft.Azure.Functions.Extensions" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" VersionOverride="3.0.37"/>
```

Comment:
Please add a tracking issue to remove any overrides that remain in the PR.   We generally do not want these to be permanent.

Reply:
Tracking issue added #48384